ISSUE_TEMPLATE.md

```diff
@@ -1,100 +1,123 @@
 Make sure you have provided the following information:
 
- - [ ] link to your code branch cloned from rhboot/shim-review in the form user/repo@tag
- - [ ] completed README.md file with the necessary information
+ - [ ] Link to your code branch cloned from rhboot/shim-review in the form user/repo@tag
+ - [ ] Completed README.md file with the necessary information
  - [ ] shim.efi to be signed
- - [ ] public portion of your certificate(s) embedded in shim (the file passed to VENDOR_CERT_FILE)
- - [ ] binaries, for which hashes are added do vendor_db ( if you use vendor_db and have hashes allow-listed )
- - [ ] any extra patches to shim via your own git tree or as files
- - [ ] any extra patches to grub via your own git tree or as files
- - [ ] build logs
- - [ ] a Dockerfile to reproduce the build of the provided shim EFI binaries
+ - [ ] Public portion of your certificate(s) embedded in shim (the file passed to VENDOR_CERT_FILE)
+ - [ ] Binaries, for which hashes are added do vendor_db ( if you use vendor_db and have hashes allow-listed )
+ - [ ] Any extra patches to shim via your own git tree or as files
+ - [ ] Any extra patches to grub via your own git tree or as files
+ - [ ] Build logs
+ - [ ] A Dockerfile to reproduce the build of the provided shim EFI binaries
 
-
-###### What organization or people are asking to have this signed:
-`[your text here]`
-
-###### What product or service is this for:
-`[your text here]`
-
-###### Please create your shim binaries starting with the 15.4 shim release tar file:
-###### https://github.com/rhboot/shim/releases/download/15.4/shim-15.4.tar.bz2
-###### This matches https://github.com/rhboot/shim/releases/tag/15.4 and contains
-###### the appropriate gnu-efi source.
-###### Please confirm this as the origin your shim.
-`[your text here]`
-
-###### What's the justification that this really does need to be signed for the whole world to be able to boot it:
+-------------------------------------------------------------------------------
+### What organization or people are asking to have this signed?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### How do you manage and protect the keys used in your SHIM?
+-------------------------------------------------------------------------------
+### What product or service is this for?
```

Comment:
These first two questions are still duplicated between readme and issue template, is that intentional?

Reply:
Oh just saw the explanation in your commit message. My feeling is that it's still hard to understand why some questions are in the readme and others are in the issue template. IMO it would be better if almost everything were in the readme, because then it's tracked by git instead of just the issue tracker history. Then the issue tracker could pretty much just be a link to the shim-review tag.

Reply:
So, I agree that it would be better for submitters if everything were just in the repo. I also believe that they added the ISSUE_TEMPLATE for a reason, and I assume it's that they want those questions easily visible in the PR.

Re-reading my description, I've dropped a lot of the stuff about different ways this could be broken up in favor of "I have thoughts which we can talk about separately". The division of questions made more sense in the context of that.

I'm still pretty unsure about it, but I think I'll mostly take your suggestion here.
* I'll move all questions except org/product from issue template to readme. I'll leave the org/product because that feels like a helpful piece of context for the reviewer. 
* I'll also leave in the "Did you remember to?" checklist at the top
* I'll add a bit to the description about how I've moved all questions to the readme

Reply:
Makes sense. My assumption is that the ISSUE_TEMPLATE/README thing probably just kinda happened organically and hadn't been revisited, but we'll find out when you submit :) 

Re the product question in the issue template, I think that is pretty much always there in the [issue title](https://github.com/rhboot/shim-review/issues) anyway, but probably doesn't matter much either way.